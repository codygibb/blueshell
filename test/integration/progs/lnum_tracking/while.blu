while 5 {
	print "?"
}
