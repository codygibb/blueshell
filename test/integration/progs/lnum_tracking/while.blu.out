error@1: Incorrect_type (while, int, bool)
